.gitignore

```
@@ -0,0 +1,22 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+.env
+.prod.env
+# dependencies
+/server/node_modules
```

Don't commit your node modules!
```suggestion
node_modules
```